mscore/workspacedialog.cpp

```diff
@@ -37,8 +37,12 @@ void MuseScore::createNewWorkspace()
 
 void MuseScore::editWorkspace()
       {
-      if (!WorkspacesManager::currentWorkspace() && !WorkspacesManager::currentWorkspace()->readOnly())
+      if (!WorkspacesManager::currentWorkspace())
```

Comment:
This is logically correct now, but what's the point of splitting it out into two separate `if` statements?

Reply:
To make it clear.

Reply:
And I think the condition `if (WorkspacesManager::currentWorkspace()) should be assert, but not sure yet